esp/esdllib/esdl_def_helper.cpp

```diff
@@ -113,6 +113,58 @@ void EsdlDefinitionHelper::setTransformParams( EsdlXslTypeId xslId, IProperties
     parameters.setValue( xslId, params );
 }
 
+void removeEclHiddenStructs(IPropertyTree &depTree)
+{
+    Owned<IPropertyTreeIterator> it = depTree.getElements("*[@ecl_hide='1']");
+    ForEach(*it)
+        depTree.removeTree(&it->query());
+}
+void removeEclHiddenElements(IPropertyTree &depTree)
+{
+    Owned<IPropertyTreeIterator> it = depTree.getElements("*");
+    ForEach(*it)
+    {
+        StringArray names;
+        Owned<IPropertyTreeIterator> elements = it->query().getElements("*[@ecl_hide='1']");
+        ForEach(*elements)
+            names.append(elements->query().queryProp("@name"));
+        ForEachItemIn(i, names)
+        {
+            VStringBuffer xpath("*[@name='%s']", names.item(i));
+            it->query().removeProp(xpath);
+        }
+    }
+}
+void removeGetDataFromElements(IPropertyTree &depTree)
```

Comment:
These removeXX methods seem duplicated in tools/esdlcmd/esdlcmd_diff.cpp, is it worthwhile to share?